taxonomy/migrations/0025_industry.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.15 on 2022-10-17 06:41
+
+from django.db import migrations, models
+from taxonomy.constants import NAICS2_CODES
+
+
+def add_industry_data(apps, schema_editor):
+    """
+    Add industry data from NAICS2 codes added in the constants.
+    """
+    Industry = apps.get_model('taxonomy', 'Industry')
+    for code, industry_name in NAICS2_CODES.items():
+        Industry.objects.create(code=code, name=industry_name)
```

Comment:
Have we tested the behavior while un-applying this migration? will it be able to drop the table on un-applying this migration OR do we have to add a script `delete_industry_data ` similar to `add_industry_data `